inst/templates/ad_adlb.R

```diff
@@ -259,6 +275,46 @@ adlb <- adlb %>%
     check_type = "error"
   )
 
+# Get WBC
```

Comment:
The `derive_param_wbc_abs` function call would need to be done quite early, probably just before we create `ANRIND`.

Reply:
With [4802139](https://github.com/pharmaverse/admiral/pull/998/commits/480213993a86e5a38c70da28d253238461bc8d5f) the `derive_param_wbc_abs ` function call is before we create ANRIND.